lib/scarpe/document_root.rb

```diff
@@ -30,7 +30,7 @@ def do_js_eval(js)
     end
 
     def empty
-      "<body id='body-#{html_id}'><div id='wrapper-#{html_id}'></div></body>"
+      "<body id='body-#{html_id}' style='border:0;margin:0;'><div id='wrapper-#{html_id}'></div></body>"
```

Comment:
If this is merged before mine, I'll need to update for this (and the height-adjust in app.rb). If mine is merged first, feel free to grab me and I can do the merge or help you do so.